Documentation/operations/performance/scalability/identity-relevant-labels.rst

```diff
@@ -25,74 +25,81 @@ following exceptions:
+presence of the ``!`` character.
 
 Label configurations that do not contain the ``!`` character are *inclusive
-labels*. Once at least one inclusive label is added, only labels that match the
-inclusive label configuration may be considered relevant for identities.
-Additionally, when at least one inclusive label is configured, the following
-inclusive labels are automatically added to the configuration:
+label patterns*. Once at least one inclusive label pattern is added, only
+labels that match the inclusive label configuration may be considered relevant
+for identities. Additionally, when at least one inclusive label pattern is
+configured, the following inclusive label patterns are automatically added to
+the configuration:
 
-====================================== =====================================================
-Label                                  Description
--------------------------------------- -----------------------------------------------------
-``reserved:.*``                        Include all ``reserved`` labels
-``any:io.kubernetes.pod.namespace``    Include all ``io.kubernetes.pod.namespace`` labels
-``any:io.cilium.k8s.namespace.labels`` Include all ``io.cilium.k8s.namespace.labels`` labels
-``any:app.kubernetes.io``              Include all ``app.kubernetes.io`` labels
-====================================== =====================================================
+========================================== =====================================================
+Label                                      Description
+------------------------------------------ -----------------------------------------------------
+``reserved:.*``                            Include all ``reserved:`` labels
+``any:io\.kubernetes\.pod\.namespace``     Include all ``io.kubernetes.pod.namespace`` labels
+``any:io\.cilium\.k8s\.namespace\.labels`` Include all ``io.cilium.k8s.namespace.labels`` labels
+``any:app\.kubernetes\.io``                Include all ``app.kubernetes.io`` labels
+========================================== =====================================================
 
 
 
 Configuring Identity-Relevant Labels
 ------------------------------------
 
 To limit the labels used for evaluating Cilium identities, edit the Cilium
-ConfigMap object using ``kubectl edit cm -n kube-system cilium-config``
-and insert a line to define the labels to include or exclude. Alternatively,
+ConfigMap object using ``kubectl edit cm -n kube-system cilium-config`` and
+insert a line to define the label patterns to include or exclude. Alternatively,
 this attribute can also be set via helm option ``--set labels=<values>``.
 
-
 .. code-block:: yaml
 
     apiVersion: v1
     data:
     ...
       kube-proxy-replacement: partial
-      labels:  "k8s:io.kubernetes.pod.namespace k8s:k8s-app k8s:app k8s:name"
+      labels:  "k8s:io.kubernetes\\.pod\\.namespace k8s:k8s-app k8s:app k8s:name"
```

Comment:
Why are we escaping the `\\`?

Reply:
ah, it might not be obvious on a first sight. Can we add a comment here? Otherwise people will assume it is a mistake and will remove one of the `\`.

Reply:
Good point. Done.